src/Hintman/Config.hs

```diff
@@ -0,0 +1,43 @@
+module Hintman.Config
+    ( HintmanConfig(..)
+    , defaultHintmanConfig
+    , loadFileConfig
+    ) where
+
+import Toml (AnyValue (..), BiMap (..), BiToml, (.=))
+
+import qualified Toml
+
+
+
```

Comment:
According to our style guide, there should be 2 empty lines after import section, not 1 :slightly_smiling_face: